### DISCUSSION TITLE
issue :  invalid reference format.

### DISCUSSION THREAD

**Discussion by hmakumbu** [milvus-io/milvus#34587] [Q&A and General discussion] [answered]
Hi,  I hope your are well.  

I wanted just to contact about my issue,  When I execute my script bash standalone_embed.sh start, I find this error in the terminal :    

docker: invalid reference format. See 'docker run --help'. Start failed.

**Comment by yhmo** [comment]
Double-check the docker version matches the prerequisite: https://milvus.io/docs/prerequisite-docker.md

If still fails, try downloading the milvus image by this command: `docker pull milvusdb/milvus:v2.4.5`

**Reply by hmakumbu** [reply]
I have still error, when I tried this command : docker pull milvusdb/milvus:v2.4.5.... I got this error : 
What's next:
    View a summary of image vulnerabilities and recommendations → docker scout quickview milvusdb/milvus:v2.4.5
Error response from daemon: Head "https://registry-1.docker.io/v2/milvusdb/milvus/manifests/v2.4.5": unauthorized: incorrect username or password

**Reply by hmakumbu** [reply, marked as answer]
I solved the problem,  I should just change the folder
